ui/lib/ldap/addon/engine.js

```diff
@@ -5,7 +5,7 @@
 
 import Engine from 'ember-engines/engine';
 import loadInitializers from 'ember-load-initializers';
-import Resolver from './resolver';
+import Resolver from 'ember-resolver';
```

Comment:
I removed the resolver that was generated when setting up the engine since it only imported from `ember-resolver` and re-exported it but forgot to update the import here.